data-conversion/src/main.rs

```diff
@@ -252,3 +267,43 @@ fn fixed_to_binary(
 
     Ok(())
 }
+
+fn binary_to_fixed(
```

Comment:
Please consider writing a detailed documentation string for each function to explain exactly what it does. It would be best if you could give examples of what the arguments should be… for example, it's not clear (just from reading the argument names) what `binary_string` is supposed to contain. I think it's supposed to be an ASCII string of `1` and `0` characters representing a double-precision floating-point number?